client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+
+export default React.createClass( {
+	displayName: 'BluehostStoreStep',
+
+	redirectToPartner() {
+		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+		} else {
+			window.location.href = 'https://www.bluehost.com/wordpress';
+		}
```

Comment:
Are we really sure we want to do this here? I'm pretty sure we have a specific way to redirect ,either through the `page` middleware or by some Redux action. Let's be very careful about introducing globals like `window` in React components because it can break tests and prevent server-side rendering.